templates/next/pages/foos/[id]/index.tsx

```diff
@@ -1,30 +1,63 @@
-import { NextComponentType, NextPageContext } from 'next';
-import { Show } from '../../../components/{{{lc}}}/Show';
-import { {{{ucf}}} } from '../../../types/{{{ucf}}}';
-import { fetch } from '../../../utils/dataAccess';
+import { GetStaticPaths, GetStaticProps, NextComponentType, NextPageContext } from "next";
+import { Show } from "../../../components/{{{lc}}}/Show";
+import { {{{ucf}}} } from "../../../types/{{{ucf}}}";
+import { fetch } from "../../../utils/dataAccess";
 import Head from "next/head";
+import DefaultErrorPage from "next/error";
+import { useMercure } from "../../../utils/mercure";
 
 interface Props {
   {{{lc}}}: {{{ucf}}};
+  hubURL: string;
```

Comment:
```suggestion
  hubURL: ?string;
```

Mercure support can be disabled.